src/Schema/SchemaContainer.php

```diff
@@ -171,10 +169,33 @@ protected function hasPath(Connection $connection, $parent)
     }
 
     /**
+     * Add connection to collection.
+     *
+     * @param  string $name
+     * @param  string $namespace
+     * @return Field
+     */
+    public function connection($name, $namespace)
+    {
+        $edgeType = $this->createField($name.'Edge', $namespace);
+
+        $this->types->push($edgeType);
+
+        $connectionType = $this->createField($name.'Connection', $namespace);
```

Comment:
I think I'm confused. The way I implemented this when I was redoing it was `Relay::connection('posts', 'PostsConnection');`

So, the generated types are `postsConnection` and `postsEdge`. If they said their type was `GraphQL::type('posts')` for their connection, it wouldn't work because it wouldn't give them the connection type, right?